tests/test_utils/test_mixins.py

```diff
@@ -1,13 +1,16 @@
 import pytest
 
-from aiogram.utils.mixins import ContextInstanceMixin, DataMixin
+from aiogram.utils.mixins import (
+    ContextInstanceMixin,
+    DataMixin,
+)
 
 
-class DataObject(DataMixin):
+class ContextObject(ContextInstanceMixin["ContextObject"]):
```

Comment:
Just use postponed evaluation of annotations (`from __future__ import annotations`)